js/src/tools/sanitizer.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * --------------------------------------------------------------------------
+ * Bootstrap (v4.3.0): tools/sanitizer.js
```

Comment:
Why not just add this to the existing https://github.com/twbs/bootstrap/blob/v4-dev/js/src/util.js?

Is it to prevent the potential bloat of individual plugin files that don't require this?

Reply:
Yep, it adds a lot of code and it's just for two plugins

Reply:
Fair enough 😄